samples/bot-conversation/python/.vscode/launch.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Add extensions.json
Refer to https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/tab-personal-quickstart/ts/.vscode/extensions.json

Reply:
added